test/integration/test_bulk_updates.py

```diff
@@ -0,0 +1,116 @@
+import time
+from unittest import TestCase
+
+import requests
+from ingest.api.ingestapi import IngestApi
+from ingest.downloader.data_collector import DataCollector
+from ingest.downloader.downloader import XlsDownloader
+from ingest.importer.importer import XlsImporter
+from openpyxl.worksheet.worksheet import Worksheet
+
+from broker.submissions import ExportToSpreadsheetService
+
+# TODO test the following flow:
+# import new spreadsheet -> download spreadsheet -> update spreadsheet -> import updated spreadsheet
+
+
+class BulkUpdateTest(TestCase):
+    def setUp(self) -> None:
+        # TODO add configuration to get ingest url from env variables
+        # TODO currently it is hardcode for dev API
+        self.ingest_url = "https://api.ingest.dev.archive.data.humancellatlas.org"
+        self.ingest_api = IngestApi(self.ingest_url)
+        # self.ingest_api = IngestApi()
+
+        self.export_service = ExportToSpreadsheetService(self.ingest_api)
+
+        self.importer = XlsImporter(self.ingest_api)
+        self.submission_url = None
+        self.data_collector = DataCollector(self.ingest_api)
+        self.downloader = XlsDownloader()
+
+    def test_download_spreadsheet(self):
+        # download the test spreadsheet for the new submission
+
+        # create a submission, track the submission url
+        self.submission_url = 'http://localhost:8080/submissionEnvelopes/60f570c9a8a292649e814171'
+
+        # download a spreadsheet
+        entity_list = self.data_collector.collect_data_by_submission_uuid('98ec9cbb-e76c-4f8f-9b97-98fce7b151cb')
```

Comment:
The test should not rely on a project in ingest db. It should be independent of the state of the DB on dev environment, or on anything external.
Specifically here, the spreadsheet should be part of the test files.

Reply:
That has been refactored